src/pudl/analysis/record_linkage/eia_ferc1_train.py

```diff
@@ -213,35 +217,18 @@ def _prep_eia_ferc1(eia_ferc1, utils_eia860) -> pd.DataFrame:
     return eia_ferc1_prep
 
 
-def _prep_ppe(ppe, utils_eia860) -> pd.DataFrame:
+def _prep_ppe(ppe: pd.DataFrame) -> pd.DataFrame:
     """Prep PPE table for use in override output sheet pre-utility subgroups.
 
     Args:
-        ppe (pd.DataFrame): The dataframe resulting from pudl_out.plant_parts_eia
-        utils_eia860 (pd.DataFrame): The dataframe resulting from pudl_out.utils_eia860.
+        ppe: The :ref:`out_eia__yearly_plant_parts` table as a dataframe.
 
     Returns:
-        pd.DataFrame: A version of the plant_parts_eia table that's been modified for
-            the purposes of creating an manual mapping spreadsheet.
+        A version of the plant_parts_eia table that's been modified for the purposes of
+        creating an manual mapping spreadsheet.
     """
     logger.debug("Prepping Plant Parts Table")
-
-    # Add utilty name eia and only take relevant columns
-    """
-    ppe_out = (
-        ppe.reset_index()
-        .merge(
-            utils_eia860[["utility_id_eia", "utility_name_eia", "report_date"]].copy(),
-            on=["utility_id_eia", "report_date"],
-            how="left",
-            validate="m:1",
-        )[RELEVANT_COLS_PPE]
-        .copy()
-    )
-    """
```

Comment:
Removed this commented out code. @cmgosnell do you happen to know why it was hanging around?